files/service_files/disable_thp.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
can you add a comment at the top like `# THIS FILE IS MANAGED BY PUPPET`? Same for the .timer file

Reply:
Correct, it's [`/usr/bin/hugeadm`](https://packages.debian.org/buster/amd64/libhugetlbfs-bin/filelist) on Debian, and it's not Arch where `/bin` is symlinked.